web_api/app.py

```diff
@@ -0,0 +1,131 @@
+import json
+
+from werkzeug.utils import secure_filename
+from flask import Flask, jsonify, request
+import os
+import uuid
+from datetime import datetime
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/